pkg/kubelet/apis/kubeletconfig/validation/validation.go

```diff
@@ -100,5 +100,13 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration) error
 				val, kubetypes.NodeAllocatableEnforcementKey, kubetypes.SystemReservedEnforcementKey, kubetypes.KubeReservedEnforcementKey))
 		}
 	}
+	switch kc.HairpinMode {
+	case kubeletconfig.HairpinNone:
+	case kubeletconfig.HairpinVeth:
+	case kubeletconfig.PromiscuousBridge:
```

Comment:
What are we trying to do here? This will fall back to default section for any given value. I think we should have `continue` here to skip the error for valid values.

Reply:
> This will fall back to default section for any given value. I think we should have continue here to skip the error for valid values.

I DON'T think so. 

Go's switch is like the one in C, C++, Java, JavaScript, and PHP, **except that Go only runs the selected case, not all the cases that follow**. 

**In effect, the break statement that is needed at the end of each case in those languages is provided automatically in Go**.